### PR TITLE
Create rent's model

### PR DESCRIPTION
**Pull request by gussiciliano**
### Sumary
- create a rent's model with a referece to User and Book

### Trello Card
https://trello.com/c/fXiTokZ7/22-crear-modelo-de-rent

### CODE REVIEW

File: app/models/rent.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Rent < ApplicationRecord
+
+  validates :from, :to, presence: true
+
+  belongs_to :user
+  belongs_to :book
+
+end
```
Comment (Dinuuu) [inline app/models/rent.rb]: Its convention to add a new line at the end of each file

File: app/models/rent.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Rent < ApplicationRecord
+
+  validates :from, :to, presence: true
+
+  belongs_to :user
```
Comment (Dinuuu) [inline app/models/rent.rb]: shouldn't this be also obligatory ?

File: app/models/rent.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  belongs_to :book
```
Comment (Dinuuu) [inline app/models/rent.rb]: idem

File: app/models/rent.rb (hunk 2 above)
Comment (gussiciliano) [inline app/models/rent.rb]: Oks

File: app/models/rent.rb (hunk 3 above)
Comment (gussiciliano) [inline app/models/rent.rb]: Oks

File: app/models/rent.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+class Rent < ApplicationRecord
+
+  validates :from, :to, presence: true
+
+  belongs_to :user, presence: true
```
Comment (Dinuuu) [inline app/models/rent.rb]: Check if this works, if not, move it to validates

**Pull request merged**